chapter3.adoc

```diff
@@ -54,6 +60,17 @@ A dedicated error code, security fault error (cmderr 6), is included in `cmderr`
 
 The error raised by resethaltreq, reset can be identified through the fields `allsecfault` and `anysecfault` in dmstatus. Error status bits are internally maintained for each hart, with the `allsecfault` and `anysecfault` fields indicating the error status of the currently selected harts. These error statuses are sticky and can only be cleared by writing 1 to `acksecfault` in dmcs2.
 
+=== Non-secure Debug control
+
+The state element `nsecdbg` is introduced to retain full debugging capabilities, as if the extensions in this specification were not implemented. When `nsecdbg` is set to 1:
+
+* All debug operations are executed with M-mode privilege (equivalent to having `mdbgen` set to 1) for all harts in the system.
+* The ndmreset operation is allowed.
+* The `relaxedpriv` field becomes configurable.
```

Comment:
An implementation may opt to hardcode relaxedpriv regardless of Sdsec.  I would say that "`relaxedpriv` may be configurable."